docs/developers/setup-local-testnet.md

```diff
@@ -14,29 +14,29 @@ The mini-testnet contains:
 
 If not specified otherwise, the mini-testnet starts with one Shard plus the Metachain (each with one Validator and one Observer).
 
-## **Prerequisites: erdpy**
+## **Prerequisites: mxpy**
 
-In order to install erdpy, follow the instructions at [install erdpy](/sdk-and-tools/erdpy/installing-erdpy#install-using-erdpy-up-recommended).
+In order to install mxpy, follow the instructions at [install mxpy](/sdk-and-tools/mxpy/installing-mxpy#install-using-mxpy-up-recommended).
 
-:::tip erdpy version
-Make sure your erdpy version is `1.3.2` or higher.
+:::tip mxpy version
+Make sure your mxpy version is `1.3.2` or higher.
```

Comment:
I'd say we can increase the mxpy version here, as we are close to launching `mxpy v5.0.0`

Reply:
Indeed. Fixed.